packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -454,9 +454,6 @@ class FlutterWebPlatform extends PlatformPlugin {
         } on FormatException catch (ex) {
           _logger.printError('Caught FormatException: $ex');
           return shelf.Response.ok('Caught exception: $ex');
-        } on IOException catch (ex) {
```

Comment:
Why are we not catching this here?

Reply:
This catch was introduced recently, but before I introduced `getChromeTabGuarded`. `getChromeTabGuarded` should already catch this.